Core/include/Acts/Seeding/PathSeeder.hpp

```diff
@@ -9,8 +9,18 @@
 #pragma once
 
 #include "Acts/EventData/SourceLink.hpp"
+#include "Acts/EventData/TrackParameters.hpp"
+#include "Acts/Seeding/detail/UtilityFunctions.hpp"
 #include "Acts/Surfaces/Surface.hpp"
 #include "Acts/Utilities/Delegate.hpp"
+#include "Acts/Utilities/GridIterator.hpp"
+
+namespace {
+
+template <typename grid_t>
+concept SourceLinkGrid =
+    std::same_as<typename grid_t::value_type, std::vector<Acts::SourceLink>>;
+}
```

Comment:
Anonymous namespace is not super helpful in headers. Can you nest it somewhere, or put it in a named namespace `detail`?

Reply:
Yes, it is geometry independent. Removed `Experimental`

Reply:
Put it together with other concepts for seeding